### PR TITLE
Lab 5 6 branch

### PR DESCRIPTION
**Pull request by DmtkD**
I have created a unit test for class StoreManager

### CODE REVIEW

File: Lab2/Lab2.iml [hunk 1]
```diff
@@ -7,9 +7,108 @@
       <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
```
Comment (redd4ford) [inline Lab2/Lab2.iml:7]: You really don't need to import *.iml, as it is a config file created by your IDE. This one will be recreated if someone clones and opens your project with the same IDE, and may cause problems for those who use any other IDE (for example Eclipse)

File: Lab2/src/main/java/ua/lviv/IoT/lab2/model/Chemicals.java [hunk 2]
```diff
@@ -14,6 +14,15 @@ public Chemicals(double price, double volume, String company, boolean forCleanin
       this.forCleaning = forCleaning;
    }
 
+
+   public String getHeader() {
+      return String.format("%s, %s, %s, %s", "price", "volume", "company", "forCleaning");
+   }
+
+   public String toSVC() {
```
Comment (redd4ford) [inline Lab2/src/main/java/ua/lviv/IoT/lab2/model/Chemicals.java]: I believe you meant CSV? CSV stands for comma separated values.
There are the same methods for each model, so I'm not going to write separate comments, it's just a common typo for all of them.

File: Lab2/src/main/java/ua/lviv/IoT/lab2/manager/impl/ChemicalsWriter.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package ua.lviv.IoT.lab2.manager.impl;
+
+import ua.lviv.IoT.lab2.manager.IChemicalsWriter;
+import ua.lviv.IoT.lab2.model.Chemicals;
```
Comment (redd4ford) [inline Lab2/src/main/java/ua/lviv/IoT/lab2/manager/impl/ChemicalsWriter.java]: Class names should be singular nouns: Chemical, Detergent, Kitchen, Toilet.

**Review by redd4ford — CHANGES_REQUESTED** [review]
I can see you added Checkstyle but I believe you didn't actually check anything with it. If you run `mvn site` in your terminal, it will generate you a checkstyle.html file in the target\site folder. There, you can see all of the problems with your code.
Do not fix stuff that messes up your logic/design (e.g. visibility modifiers, "should be final", DesignForExtension stuff). Also, do not fix "Missing a Javadoc comment", "Missing package-info.java file" (or anything javadoc-related) and "NUMBER is a magic number". But LineLength, NewlineAtEndOfFile, ParameterName, AvoidStarImport are a must-have.

File: Lab2/src/main/java/ua/lviv/IoT/lab2/model/Chemicals.java (hunk 2 above)
Comment (DmtkD) [reply Lab2/src/main/java/ua/lviv/IoT/lab2/model/Chemicals.java]: Yes, It was just my mistake. I have fixed it.

File: Lab2/src/main/java/ua/lviv/IoT/lab2/manager/impl/ChemicalsWriter.java (hunk 3 above)
Comment (DmtkD) [reply Lab2/src/main/java/ua/lviv/IoT/lab2/manager/impl/ChemicalsWriter.java]: Thank you for your comments. Also fixed it.

**Pull request merged**